setup-dev-cluster.sh

```
@@ -12,6 +12,62 @@ ingress_https_port=443
 reg_name='kind-registry'
 reg_port='5001'
 
+# Check for required dependencies
+check_dependencies() {
+  local missing_deps=()
```

Shellcheck (0.11.0)

[warning] In POSIX sh, 'local' is undefined.

(SC3043)